tests/test_bot.py

```diff
@@ -2998,7 +2998,10 @@ async def test_arbitrary_callback_data_pinned_message_reply_to_message(
         )
 
         message = Message(
-            1, None, None, reply_markup=bot.callback_data_cache.process_keyboard(reply_markup)
+            1,
+            dtm.datetime.utcnow(),
```

Comment:
can you pinpoint why passing an (admittably invalid) `None` here doesn't work anymore? 🤔

Reply:
it's because we did `data["date"] = to_timestamp(self.date)` in `Message.to_dict`. Where `to_timestamp` returned `None` if `self.date` was `None`. This means that `Message.de_json` would pass. But now we don't do that, so `Message.de_json` returns `KeyError: data["date"]`

Reply:
> But now we don't do that

Because `TO._get_attrs` skips `None` values, right? Okay, that sounds reasonable :)

Reply:
> Because TO._get_attrs skips None values, right?

Precisely